frontend/context/app.tsx

```diff
@@ -21,6 +21,7 @@ enum ACTIONS {
   SET_SANDBOX_EXPIRY = "SET_SANDBOX_EXPIRY",
   SET_NO_SANDBOX_HOSTS = "SET_NO_SANDBOX_HOSTS",
   SET_FILTERED_HOSTS_PATH = "SET_FILTERED_HOSTS_PATH",
+  SET_FILTERED_SOFTWARE_PATH = "SET_FILTERED_SOFTWARE_PATH",
```

Comment:
Just like last filtered host path ensured the back to all hosts goes back to the correct hosts table state, setting a last filtered software path ensures the back to all software goes back to the correct software table state.

Reply:
Already included in other PR that's already merged